Feat / accessibility epg alt text for images

langemike opened the pull request:
Fix alt text for EPG.

ps. EPG is broken in `feat/a11y-sprint-2 branch`. This issue is fixed in `feat/restructure-sprint-2`.

Ticket: https://videodock.atlassian.net/browse/OTT-404
ps2. Yes, this is a high-prio quickwin for accessibility!

langemike committed:
fix(a11y): alt text for images for EPG

ChristiaanScheermeijer commented on packages/ui-react/src/components/Epg/Epg.tsx:
```diff
@@ -64,19 +64,23 @@ export default function Epg({ channels, onChannelClick, onProgramClick, channel,
+                  onScrollToNow();
+                }}
+                title={channel?.title || ''}
+                isActive={!!channel}
```
This might not be clear from reading the arguments/props, but the Epg channel prop is the currently selected/active channel.

The `isActive` prop should be set to `true` for only the active channel, but currently will be set to `true` for all channels. Maybe we can rename the prop to `activeChannel` or `selectedChannel` (the term active is already used here).

langemike replied:
While i was laying on my pillow, thinking about this, I realized this too and opened my laptop to have a look at it and saw you noticed this too 37 min ago haha. I will fix this tomorrow ;)

ChristiaanScheermeijer reviewed (commented):
Nice quick win indeed! Only the active channel needs a small fix, but the rest looks good! (linting also seem to fail)

langemike commented:
@ChristiaanScheermeijer fixed the lint errors, used a dictionary (for performance) and corrected my `isActive` mistake. Also renamed the prop to make it more clear. Please re-review

langemike committed:
fix(e2e): fix attribute selector

royschut reviewed (approved):
Nice @langemike !
